### PR TITLE
Custom Targets for syncing

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

**Comment by kchitalia** [comment]
@wmathurin @bhariharan

File: SalesforceSDK/Salesforce.SDK.SmartSync/Model/SyncUpTarget.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
+﻿using System;
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.SmartSync/Model/SyncUpTarget.cs]: Add copyright.

File: SalesforceSDK/Salesforce.SDK.SmartSync/Model/SyncTarget.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿/*
- * Copyright (c) 2014, salesforce.com, inc.
+ * Copyright (c) 2015, salesforce.com, inc.
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.SmartSync/Model/SyncTarget.cs]: This should probably be ```2014-2015```.

**Comment by bhariharan** [comment]
Otherwise, looks pretty good to me.

**Comment by wmathurin** [comment]
Looks fine.

**Comment by kchitalia** [comment]
Changes made based on comments.

**Pull request merged**